### ISSUE TITLE
Support for "Liveview (15min) No Timeout" on D5200 C 1.03

### ISSUE THREAD

**Issue by moeinalmasi** [simeonpilgrim/nikon-firmware-tools#117]
Currently, there is no support for "Liveview (15min) No Timeout" for C 1.03. Is there a way to downgrade the firmware? I perhaps will try to open a PR to add the support for C 1.03 firmware. Thank you!

**Comment by ian5142** [comment]
I would like this as well. Plus, raising the video limit past 20 min, also for C 1.03.

**Comment by ian5142** [comment]
Here is my original ticket: https://github.com/simeonpilgrim/nikon-firmware-tools/issues/79

**Comment by ian5142** [comment]
And this ticket: https://github.com/simeonpilgrim/nikon-firmware-tools/issues/88

**Comment by simeonpilgrim** [comment]
> Is there a way to downgrade the firmware?

Yes, the camera can use the older firmware, just fine.

**Comment by simeonpilgrim** [comment]
> I perhaps will try to open a PR to add the support for C 1.03 firmware. Thank you!

That sounds a grand idea!

**Comment by moeinalmasi** [comment]
> > Is there a way to downgrade the firmware?
> 
> 
> 
> Yes, the camera can use the older firmware, just fine.

The camera doesn't let you to downgrade, at least not directly. I will open a PR.

**Comment by simeonpilgrim** [comment]
> The camera doesn't let you to downgrade, at least not directly. 
I am not sure what you mean by this, there is clearly a misconnect on word usage/meaning. 

> I will open a PR.
Excellent.

**Comment by moeinalmasi** [comment]
> I am not sure what you mean by this, there is clearly a misconnect on word usage/meaning.

You are right. I managed to downgrade the 1.03 to 1.02. The issue that I had was with the patched firmware not being detected by the camera (meaning when I would go to the firmware version, I could not see the update button). It is resolved by removing the "patched" from the file name. 

I have started to look at adding the no timeout to 1.03, hopefully in the upcoming weekends. Cheers!
